dask/dataframe/__init__.py

```diff
@@ -117,6 +118,7 @@ def inner_func(*args, **kwargs):
         from dask.dataframe.numeric import to_numeric
         from dask.dataframe.optimize import optimize
         from dask.dataframe.reshape import get_dummies, melt, pivot_table
+        from dask.dataframe.rolling import map_overlap
```

Comment:
Could make the `rolling.` prefix disappear from the rendered documentation?
(I can see two cases in docs/source)

![image](https://github.com/dask/dask/assets/6213168/2f497366-cb53-469d-b905-eacca2d7146f)

Reply:
done